components/toast/index.css

```diff
@@ -10,18 +10,60 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
+/* toast/index.css
+ * 
+ * .spectrum-Toast::after is the Focus ring
+ *
+ * Component naming template:
+ * `--system-component-style-description-modifier`
+ *
+ * Example:
+ * `--spectrum-toast-spacing-text-to-button-horizontal`
+ */
+
 .spectrum-Toast {
-  --spectrum-toast-icon-padding-y: var(--spectrum-global-dimension-size-85);
-  --spectrum-toast-neutral-content-padding-top: var(
-    --spectrum-global-dimension-size-65
-  );
-  --spectrum-toast-content-padding-bottom: var(
-    --spectrum-global-dimension-size-65
-  );
-
-  --spectrum-toast-button-margin-right: var(
-    --spectrum-global-dimension-size-130
-  );
+  /* Hardcoded variables */
+  --spectrum-toast-font-weight: 400;
+
+  /* Size */
+
+  --spectrum-toast-font-size: var(--spectrum-font-size-100);
```

Comment:
I think in all of these definitions we need to add the `--mod`  option as well to enable custom overwrites. 

like `  --spectrum-toast-font-size: var(--mod-font-size-100, var(--spectrum-font-size-100);
`

more details here: https://github.com/adobe/spectrum-css/wiki/Migrating-a-component-to-core-tokens#implementing-the-custom-properties-api

Reply:
Oh I added the `--mod` prefix when I use the component token e.g. `font-size: var(--mod-toast-font-size, var(--spectrum-toast-font-size));`. Is that what you mean?

Reply:
yes, just checking this is consistently used.